api/database/factories/ModelFactory.php

```diff
@@ -106,3 +107,20 @@
 
     return compact('token') + $body;
 });
+
+$factory->define(App\Models\ArticlePermalink::class, function ($faker) {
+    return [
+        'permalink' => str_random(255),
```

Comment:
Can you change this to `$faker->unique()->slug` to give more realistic permalinks (these values are used on the frontend for demonstrations)